code/game/objects/items/devices/flashlight.dm

```diff
@@ -116,7 +116,8 @@
 			return
 		var/T = get_turf(target)
 		if(locate(/mob/living) in T)
-			PoolOrNew(/obj/effect/medical_holosign, list(T,user)) //produce a holographic glow

```

Comment:
ooooooh wow that's an easy fix except you did it wrong gimme a second while I add the proper fix to my already-existing pr for overlay/temp objects #18872